runhouse/servers/http/http_server.py

```diff
@@ -115,15 +117,20 @@ async def ainitialize(
         default_env_name=None,
         conda_env=None,
         from_test: bool = False,
+        disable_observability: bool = False,
+        logs_level: str = DEFAULT_LOG_LEVEL,
```

Comment:
minor point but seems like some of these log changes are kinda separate from the observability changes.. is there another reason why they are included in this PR?